src/campaign.cairo

```diff
@@ -65,6 +65,7 @@ mod TokengiverCampaign {
         #[key]
         campaign_address: ContractAddress,
         token_id: u256,
+        token_giverNft_contract_address: ContractAddress,
     }
 
```

Comment:
add a constructor function that accept token_giver_nft_class_hash and save it in storage

token_giver_nft_class_hash: ClassHash